docs/notebooks/running_test_application.ipynb

```diff
@@ -0,0 +1,354 @@
+    "from sar_pre_processing.sar_pre_processor import *\n",
+    "import warnings\n",
+    "warnings.filterwarnings(\"ignore\")\n",
+    "\n",
+    "processing = SARPreProcessor(config='test_config_file.yaml')\n",
+    "processing.create_processing_file_list()\n",
+    "print('start step 1')\n",
+    "processing.pre_process_step1()\n",
+    "print('start step 2')\n",
+    "processing.pre_process_step2()\n",
+    "print('start step 3')\n",
+    "processing.pre_process_step3()\n",
+    "print('start add netcdf information')\n",
+    "processing.add_netcdf_information()\n",
+    "print('start create netcdf stack')\n",
+    "processing.create_netcdf_stack()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 3. View processed data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Load netcdf file with processed data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "print(os.getcwd())\n",
+    "print(output_folder)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from netCDF4 import Dataset\n",
+    "import numpy as np\n",
+    "\n",
+    "my_example_nc_file = os.path.join(output_folder + 'data.nc')\n",
+    "data = Dataset(my_example_nc_file, mode='r') "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "View information about dataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Read data from netcdf file"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data.variables['orbitdirection'][:]\n",
+    "data.variables['time'][:]\n",
+    "lons = data.variables['lon'][:]\n",
+    "lats = data.variables['lat'][:]\n",
+    "vv = data.variables['sigma0_vv_multi'][:]\n",
+    "\n",
+    "vv_units = data.variables['sigma0_vv_multi'].units"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Close netcdf file"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "data.close()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Plot vv polorized data"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "try:\n",
+    "    import matplotlib\n",
+    "    print(\"matplotlib installed - requirement satisfied.\")\n",
+    "except ModuleNotFoundError:\n",
+    "    print('matplotlib module not installed!')\n",
+    "    yes = input(\"Do you want do install now? (type y)\")\n",
```

Comment:
This did not work in my installation. Also, when using conda we should avoid using pip whenever possible. Either add matplotlib as requirement in environments.yml or state at the beginning of the notebook that users should execute 'conda install -c conda-forge matplotlib' first.

Reply:
pip statement removed, conda statement included